### PR TITLE
No easy way to retrieve Raw pixel formats #94

### PR DESCRIPTION
**Pull request by afwlehmann**
Possible fix for issue #94. Let me know if this would be ok for you so that I could adapt the rest of the code and examples to match the new signatures.

### CODE REVIEW

**Comment by afwlehmann** [comment]
Done. I think this approach makes the usage of `createRGBSurface` and `createRGBSurfaceFrom` easier and more consistent in terms of a high-level API.

File: src/SDL/Video/Renderer.hs [hunk 1]
```diff
@@ -304,15 +306,15 @@ queryTexture (Texture tex) = liftIO $
 -- | Allocate a new RGB surface.
 --
 -- See @<https://wiki.libsdl.org/SDL_CreateRGBSurface SDL_CreateRGBSurface>@ for C documentation.
-createRGBSurface :: (Functor m,MonadIO m)
+createRGBSurface :: (MonadIO m)
```
Comment (ocharles) [inline src/SDL/Video/Renderer.hs]: We need that `Functor` constraint for GHC 7.8.

Comment (afwlehmann) [inline src/SDL/Video/Renderer.hs]: Ah, I see. Will fix that.

**Comment by polarina** [comment]
Thanks! Cherry-picked as 89c4f8b06798c81b56bff9aa05abe29ff9aad026 and 472f3abe8a9a1e8970b19e9f91a65f6c62a8a97d.

Apologies for the delay.

**Pull request closed without merging**